### PR TITLE
Adequating code to pep8

### LINKED ISSUE #37: Modify component code to PEP8 standards
In my opinion, the component code can be more organized with the purpose of facilitate its modifications/implementations and make it more readable. PEP8 is style guide that can help this issue to be solved. However, can exists a different and more efficient style guide that can help us top solve this problem. Please, if you have any other suggestions, comment in this issue so we can discuss and take the right choice.

### PR DESCRIPTION
**Pull request by matheuscmelo**
This PR is able to fix pep8 issues in the manager code. 
Fixes #37.

### CODE REVIEW

**Review by rafaelvfalc — CHANGES_REQUESTED** [review]
This is a great contribution, thank you guys. I will perform the necessary tests to verify the component behavior. However, I noticed that both commits has similar messages, can one of you guys squash the commits in just one (since both commits performed the same action)?

**Comment by matheuscmelo** [comment]
@rafaelvfalc done

**Comment by rafaelvfalc** [comment]
I also think that might be important to test all PEP8 versions of the components together. In this way we can correctly verify that the behavior of each component has not changed by the modifications.

**Review by rafaelvfalc — CHANGES_REQUESTED** [review]
Please solve merge conflicts.

**Review by clenimar — APPROVED** [review]

File: broker/plugins/chronos/plugin.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import time
+# import time
```
Comment (GabrielSVinha) [inline broker/plugins/chronos/plugin.py]: If this is not being used, do not comment it, remove it.

File: broker/plugins/chronos/plugin.py [hunk 2]
```diff
@@ -44,13 +45,14 @@ def start_application(self, data):
...
 
-            # Obtain arguments 
-            start_job = time.time()
-            job = data['info_plugin']['job']            
-            deadline = data['info_plugin']['qos']['deadline'] #+ start_job 
-            job_duration = data['info_plugin']['qos']['duration'] # seconds
-            desv_deadline = data['info_plugin']['qos']['desv_deadline'] # %/100 (e.g 5% --> desv_deadline=0.05 )
-            jobname = job['name'] 
+            # Obtain arguments
+            # start_job = time.time()
```
Comment (GabrielSVinha) [inline broker/plugins/chronos/plugin.py]: Again, please remove commented code but leave useful comments like in line 48

**Review by gabrielsvinha — CHANGES_REQUESTED** [review]

File: broker/plugins/chronos/plugin.py (hunk 1 above)
Comment (javanlacerda) [reply broker/plugins/chronos/plugin.py]: Done!

File: broker/plugins/chronos/plugin.py (hunk 2 above)
Comment (javanlacerda) [reply broker/plugins/chronos/plugin.py]: Done!

**Review by clenimar — APPROVED** [review]

**Review by liliarsampaio — APPROVED** [review]
that's a lot of work, thanks! lgtm :)

**Pull request merged**